Gemfile

```diff
@@ -57,6 +57,8 @@ gem 'active_skin'
 gem 'active_admin_flat_skin'
 gem 'aasm', '~> 5.0', '>= 5.0.8'
 
+gem 'turbolinks', '~> 5.2.0'
```

Comment:
why?

Reply:
i removed, i was following an implementation but i forgot to remove it, thank u.